app/src/main/java/com/kuit/kuit6android/ui/home/component/CategoryRow.kt

```diff
@@ -0,0 +1,39 @@
+package com.kuit.kuit6android.ui.home.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kuit.kuit6android.R
+import com.kuit.kuit6android.ui.home.data.CategoryData
+
+@Composable
+fun CategoryRow(modifier: Modifier = Modifier) {
+    val categoryList = listOf(
+        CategoryData(imageId = R.drawable.pork_cutlet, category = "돈까스"),
+        CategoryData(imageId = R.drawable.japanese_food, category = "일식"),
+        CategoryData(imageId = R.drawable.korean_food, category = "한식"),
+        CategoryData(imageId = R.drawable.chicken, category = "치킨"),
+        CategoryData(imageId = R.drawable.snack_food, category = "분식"),
+    )
+//    LazyRow(horizontalArrangement = Arrangement.spacedBy(space = 4.dp)) {
+//    LazyRow(horizontalArrangement = Arrangement.spacedBy(space = 18.dp)) {
+    Row(horizontalArrangement = Arrangement.spacedBy(space = 18.dp)) {
```

Comment:
```suggestion
    Row(
        modifier = Modifier
            .fillMaxWidth()
            .padding(horizontal = 27.dp),
        horizontalArrangement = Arrangement.SpaceBetween
    ) {
```
spacedBy로만 여백을 주면 기기가 가로로 길어졌을 때, 좌우 여백이 너무 클 수 있기 때문에 FillMaxWidth와 SpaceBetween 사용하는 게 더 좋을 것 같습니다!